### ISSUE TITLE
Mean Square Error Regression Convolution Net Stop Learning

### ISSUE THREAD

**Issue by MaxChu719** [vlfeat/matconvnet#716]
1.  I am trying to use the alexnet to do regression on human pose estimation using MatConvNet.

2.  When i train the net for the first two epochs, the loss is decreasing quick. But after than, the loss is not decreasing anymore and when i visualise the first feature map, it shows just random noise.

3. The code can be found in https://github.com/MaxChu719/HumanPoseEstimation.

4. I already tried using different learning rate. When the learning rate is too high, it will diverge very quickly. If the learning rate is around 0.0001, it will starting decreasing the loss for two epochs. See the figure of the loss: http://i.stack.imgur.com/zezwL.jpg

**Comment by albanie** [comment]
It might be best to ask @MaxChu719 - the author of the original code?
